App/HejDTO/Sources/HejDTO/AccountSettingsDTO.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  AccountSettings.swift
+//  
+//
+//  Created by Dominika Kokowicz on 11/03/2023.
+//
+
+import Foundation
+
+struct AccountSettingsDTO: Codable {
```

Comment:
Add a link to hejto.pl documentation for what call this DTO is for :)

Reply:
Fixed :)